src/platform/silabs/wifi/WifiInterface.h

```diff
@@ -73,6 +75,7 @@ typedef struct wfx_wifi_scan_result
     uint8_t bssid[kWifiMacAddressLength];
     uint8_t chan;
     int16_t rssi; /* I suspect this is in dBm - so signed */
+    WiFiBandEnum wiFiBand;
```

Comment:
Is the plan to clean up this structure in this PR or in a follow up? When we modify the legacy interface elements, we should refactor them as we go.

Reply:
plan is to have a follow up PR for the legacy cleanup